components/definitions/definitions.js

```diff
@@ -0,0 +1,158 @@
+import domready from '../../assets/js/domready';
+
+export const classAnchor = 'definition__anchor';
+export const classDialog = 'popup';
+export const classDialogButton = 'popup__button';
+export const classDialogDocument = 'popup__document';
+
+export const idDialogMessage = 'd-message';
+
+export const attrResponse = 'data-popup-response';
+export const attrDataDialogCall = 'data-popup-call';
+export const attrDataTitle = 'data-title';
+
+export default function() {
+  return definitionDialog();
+}
+
+export function definitionDialog() {
+  anchorsToButton();
+  buttonClick();
```

Comment:
This isn't a good name, it doesn't describe what is going on. A better one would be something like `bindButtonClickHandlers`.